docs/mobile-apps/automated-testing/espresso-xcuitest/xcuitest.md

```diff
@@ -977,6 +977,39 @@ Instructs `saucectl` to run all classes for the suite _except_ those specified h
 
 ---
 
+### `shard`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+When shard is configured as `concurrency`, saucectl automatically splits the tests into several groups (the number of groups is determined by the concurrency setting). Each group will then run as an individual job.
+
+Selectable values: `concurrency` to shard by concurrency. Remove this field or leave it empty `""` for no sharding.
+
+:::note
+When shard is set for a suite, the [class](#class) setting is automatically ignored.
+:::
+
+```yaml
+suites:
+  - name: "I am sharded"
+    shard: concurrency
+```
+---
+
+### `testListFile`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+The file containing a list of tests is used in sharding by concurrency. It's a `txt` file and each line contains a test. Click [Sharding XCUITest introduction](./xcuitest-introduction.md#sharding-xcuitest) to see how to generate this file.
+
+```
+test1
+test2
+test3
```

Comment:
I know I listed this as an example myself in the Jira ticket, but for external purposes (and easier understanding), it'd be good to use a more descriptive approach. Since these are classes, and also methods, right?, it'd be good to list them as such here. Just more illustrative of how it's supposed to look like in the end.